core/store/src/trie/mem/iter.rs

```diff
@@ -72,6 +72,7 @@ impl<'a> Crumb<'a> {
 
 pub struct MemTrieIterator<'a> {
     root: Option<MemTrieNodePtr<'a, STArenaMemory>>,
+    trie: &'a Trie,
```

Comment:
Now that we pass in the whole trie, should we remove the value_getter function and just use the trie?